.eslintrc

```diff
@@ -4,7 +4,7 @@
   "plugins": ["import"],
 
   "env": {
-    "browser": false,
+    "browser": true,
```

Comment:
Maybe in another PR no ? Not that important IMO and it will need an update of dependencies anyway, along with a cleaner eslint configuration (airbnb, etc)